src/pages/GetStarted/GetStartedResults.vue

```diff
@@ -1,39 +1,121 @@
 <template>
 	<div>
 		<kv-progress-bar value="100" max="100" />
-		<div class="row page-content">
-			<div class="small-12 ">
-				<h1>Here are X people you can support!</h1>
-				<p>
-					In fact, we found X people that we think you’ll love to lend to.
-					<!-- <router-link>View all loans</router-link> -->
-				</p>
-				<div>loans here</div>
-
-				<section>
-					<h2>Looking for different loans?</h2>
-				</section>
-
-				<section>
-					<h2>Here's how it works</h2>
-				</section>
-
-				<section>
-					<frequently-asked-questions />
-				</section>
-			</div>
+		<div class="row column page-content">
+			<h1>Here are X people you can support!</h1>
+			<p>
+				In fact, we found X people that we think you’ll love to lend to.
+				<!-- <router-link>View all loans</router-link> -->
+			</p>
+			<div>loans here</div>
+
+			<section>
+				<h2>Looking for different loans?</h2>
+			</section>
+
+			<section class="how-it-works section text-center">
+				<div class="row">
+					<h2 class="how-it-works__header small-12 columns">
```

Comment:
since this is always spans full width I don't think it needs the wrapping .row div and .small-12 .columns